packages/common/src/web3-toolkit.test.ts

```diff
@@ -49,7 +48,11 @@ describe('Web3Toolkit', function () {
       it(`${toNetworkId(chainId)}: ${expected.symbol}`, async function () {
         const web3Toolkit = new Web3Toolkit(chainId);
         const token = await web3Toolkit.getToken(tokenAddress);
-        expect(JSON.stringify(token)).to.eq(JSON.stringify(expected));
+        expect(token.chainId).to.eq(expected.chainId);
+        expect(token.address).to.eq(expected.address);
+        expect(token.decimals).to.eq(expected.decimals);
+        expect(token.symbol).to.eq(expected.symbol);
+        expect(token.name).to.eq(expected.name);
```

Comment:
Can we keep to use `JSON.stringify` but modify the above `expected`?

Reply:
fixed with `JSON.stringify`